bindings/python/lexer.i

```diff
@@ -0,0 +1,22 @@
+%include "common.i"
+%include "fsm.i"
+
+%{
+#include <pire/re_lexer.h>
+using Pire::Lexer;
+%}
+
+class Lexer {
+public:
+    Lexer();
+
+    %extend {
+        Lexer(const char* begin, const char* end) {
+            Lexer* result = new Lexer(begin, end);
+            result->AddFeature(Pire::Features::AndNotSupport());
+            return result;
```

Comment:
Is there a memory leak in this code?

  * What if `AddFeature` throws? (Can be fixed with smart pointer with final `.release()`.)
  * Please make sure that `Lexer` object is deleted by Python.